bucket/caffeine.json

```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://www.zhornsoftware.co.uk/caffeine/index.htm",
+    "description": "Prevent PC standby or sleep",
+    "version": "1.64",
+    "license": "Unknown",
+    "url": "https://www.zhornsoftware.co.uk/caffeine/caffeine.zip#/dl.7z",
```

Comment:
Why this rename into 7zip?

Reply:
I was basing this off my altools manifest, which is a self-extracting exe. I missed this because it still works, but I'll submit a clean PR.